hub/core/storage/__init__.py

```diff
@@ -0,0 +1,4 @@
+from hub.core.storage.s3.s3 import S3Provider
```

Comment:
imports here are weird, can you do this in `__init__.py`? so you can just do `from hub.core.storage.s3 import S3Provider`